servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/GrpcClients.java

```diff
@@ -103,6 +106,21 @@ private GrpcClients() {
         return new DefaultGrpcClientBuilder<>(HttpClients.forResolvedAddress(address));
     }
 
+    /**
+     * Creates a {@link GrpcClientBuilder} for an address with default {@link LoadBalancer} and DNS {@link
+     * ServiceDiscoverer}.
```

Comment:
Because it deals with the resolved address, does it make sense to mention DNS ServiceDiscoverer here?
Some question for other `forResolvedAddress` overloads here and in `HttpClients`.

Reply:
agreed, I don't think we need to reference DNS for "resolved" address methods.